### DISCUSSION TITLE
How to add a footer link to the HC homepage template?

### DISCUSSION THREAD

**Discussion by normanwankowski** [mozilla/hubs#4981] [Hubs Cloud Questions] [answered]
I like to know what is the best way for me to add additional links to my footer on my HC homepage ?

**Comment by matthewbcool** [comment, marked as answer]
You can make edits to [`Footer.js`](https://github.com/mozilla/hubs/blob/9173a877e351dfc144a36c16230691913bb07c33/src/react-components/layout/Footer.js) on the client.

**Reply by normanwankowski** [reply]
Can you tell me please more precisely where I can find this footer.js ? Not sure where exactly where to search for it.

**Reply by normanwankowski** [reply]
Do I have to use Github for that? Like described here https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html

**Reply by matthewbcool** [reply]
correct, this requires deploying a custom client. Editing that file and deploying your version of the client to your hubs cloud gives you the most control over the footer component. Some of those links can be edited via the admin panel but you won't be able to add links and make major edits without code changes.
![image](https://user-images.githubusercontent.com/4493657/147704712-f57e1ad2-01e5-48c8-a03c-315807b50ee7.png)

**Reply by normanwankowski** [reply]
Great! will do that.

**Reply by alfainfotech** [reply]
do you know how to access code ? I deployed on AWS but when I tried to install FTP its not accessible ? anyone here accessed code on AWS for Hubs cloud
